apps/web-client/app.js

```diff
@@ -34,13 +35,16 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use((req, res, next) => {
   req.deps = {
     request,
+    url,
     photoApiUrl: `http://localhost:${process.env.API_PORT}`,
     s3Bucket: process.env.S3_BUCKET,
   };
   next();
 });
 
 // Routes
-app.use('/', require('./middleware/homepage'));
+app.get('/', require('./middleware/homepage'));
+
+app.post('/photo', require('./middleware/upload'));
```

Comment:
this isn't something that needs to change, but I usually don't think of end route code as "middleware"? I've always stored it in a folder called "routes" or something like that. Not sure it makes much of a difference, though, and it's technically correct.